cluster-install/install.sh

```diff
@@ -13,6 +13,9 @@ _PROBLEMS=() # Any failed checks will be added to this array and the program wil
 _CONFIG_BUILDER=() # Contains the fully resolved config to be output at the end during a dry run
 _KUBERNETES_VERSION=""
 
+# _TRIDENT_COLLECTION_STEP_CALLED is used as a guardrail to prevent certain functions from being called before
+# existing Trident information (if any) has been collected.
+_TRIDENT_COLLECTION_STEP_CALLED="false"
```

Comment:
This gets set to `true` during `step_collect_existing_trident_info`, and, as the comment mentions, is used to prevent devs from calling certain functions before trident info's been collected as they would've returned bogus values without raising any errors.